benchmarks/diffusion_2D/main_arkode.cpp

```diff
@@ -525,16 +616,22 @@ void UserOptions::help()
   cout << "Integrator command line options:" << endl;
   cout << "  --rtol <rtol>           : relative tolerance" << endl;
   cout << "  --atol <atol>           : absolute tolerance" << endl;
+  cout << "  --controller <ctr>      : time step adaptivity controller" << endl;
+  cout << "  --fixedstep <step>      : used fixed step size" << endl;
+  cout << "  --explicit              : use LSRKStep (instead of ARKStep)" << endl;
```

Comment:
It would be more clear if the option `--explicit` was named something like `--ls-explicit` to distinguish from regular explicit RK (if that is ever added as an option)